installers/ansible/roles/pgo-operator/tasks/main.yml

```diff
@@ -82,7 +82,7 @@
           failed_when: false
 
         - name: Create PGO Admin Credentials
-          command: "{{ kubectl_or_oc }} create -f {{ output_dir }}/pgouser-admin.yaml"
+          command: "{{ kubectl_or_oc }} create --validate=false -f {{ output_dir }}/pgouser-admin.yaml"
```

Comment:
Are we creating an invalid secret?

Reply:
Removed with Ansible fix. Thanks for the help